cloudify_rest_client/cluster.py

```diff
@@ -0,0 +1,149 @@
+########
+# Copyright (c) 2016 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+
+from cloudify_rest_client.executions import Execution
+from cloudify_rest_client.responses import ListResponse
+
+
+class ClusterNode(dict):
+    @property
+    def master(self):
+        return self.get('master', False)
+
+    @property
+    def online(self):
+        return self.get('online', False)
+
+    @property
+    def host_ip(self):
+        return self.get('host_ip')
+
+
+class ClusterState(dict):
+    @property
+    def initialized(self):
+        return self.get('initialized', False)
+
+    @property
+    def encryption_key(self):
+        try:
```

Comment:
whoops! it used to be nested before so that was cleaner. I removed the nesting but forgot to replace this :)